docs/stacks-guide-aws.md

```diff
@@ -156,693 +175,444 @@ secret/aws-user-creds created
-EOF
+As you probably have noticed, some resources are referencing other resources in
+their YAML representations. For instance for `Subnet` resource we have:
 
-# create db subnet group object in Crossplane, and wait until the corresponding
-# DB Subnet Group in AWS is created and Ready to use
-apply_and_wait_until_ready "dbsubnetgroup.yaml"
+```yaml
+...
+    vpcIdRef:
+      name: sample-vpc
+...
 ```
 
-Sample output:
+Such cross resource referencing is a Crossplane feature that enables managed
+resources to retrieve other resources attributes. This creates a *blocking
+dependency*, preventing the dependent resource from being  created before the referred
+resource is ready. In the example above, `Subnet` will be blocked until the
+referred `VPC` is created, and then it retrieves its `vpcId`. For more
+information, see [Cross Resource Referencing][].
 
-```bash
-dbsubnetgroup.storage.aws.crossplane.io/aws-network-config-dbsubnetgroup created
-dbsubnetgroup.storage.aws.crossplane.io/aws-network-config-dbsubnetgroup condition met
-```
+## Configure Resource Classes
 
-We need to retrieve the SubnetIDs so other resources can use them:
+Once we have the network configuration set up, we need to tell Crossplane how to
+satisfy WordPress's claims (that will be created when we later install the
+WordPress stack) for a database and a Kubernetes cluster. The [Resource
+Classes][resource-claims-and-classes-docs] serve as templates for the
+corresponding resource claims.
 
-```bash
-export RDS_SUBNET_GROUP_NAME=$(kubectl get -f "dbsubnetgroup.yaml" -o=jsonpath='{.spec.groupName}')
-```
+In this guide, we will use the [sample AWS resource classes][]in Crossplane
+repository.
 
-### Cluster IAM Role
+### TL;DR
 
-An [IAM Role][] gives permissions to the principal which assumes that role. We
-create a role to be assumed by the cluster, which later is granted the
-permissions needed to talk to our resources in AWS.
+Apply the sample AWS resource classes:
 
 ```bash
-# build vpc yaml
-cat > iamrole.yaml <<EOF
-apiVersion: identity.aws.crossplane.io/v1alpha2
-kind: IAMRole
-metadata:
-  name: ${CONFIG_NAME}-eks-cluster-role
-  namespace: ${INFRA_NAMESPACE}
-spec:
-  roleName: ${CONFIG_NAME}-eks-cluster-role
-  description: a role that gives a cool power
-  assumeRolePolicyDocument: |
-    {
-      "Version": "2012-10-17",
-      "Statement": [
-        {
-          "Effect": "Allow",
-          "Principal": {
-            "Service": "eks.amazonaws.com"
-          },
-          "Action": "sts:AssumeRole"
-        }
-      ]
-    }
-  reclaimPolicy: Delete
-  providerRef:
-    name: aws-provider
-    namespace: ${INFRA_NAMESPACE}
-EOF
-
-# create an IAM Role object in Crossplane, and wait until the corresponding
-# IAM Role in AWS is created and Ready to use
-apply_and_wait_until_ready "iamrole.yaml"
+kubectl apply -k github.com/crossplaneio/crossplane//cluster/examples/workloads/kubernetes/wordpress/aws/resource-classes?ref=v0.4.0
```

Comment:
remove extra `/`

Reply:
that is actually required for `kubectl apply -k` to work